src/identity/contactMapping.test.ts

```diff
@@ -146,3 +154,58 @@ describe('Fetch Addresses Saga', () => {
     })
   })
 })
+
+describe('Fetch Address Verification Saga', () => {
+  describe('central lookup', () => {
+    beforeEach(() => {
+      mockFetch.resetMocks()
+    })
+
+    it('fetches and stores verified address', async () => {
+      mockFetch.mockResponseOnce(JSON.stringify({ data: { addressVerified: true } }))
+
+      await expectSaga(fetchAddressVerificationSaga, fetchAddressVerification(mockAccount))
+        .provide([
+          [select(verifiedAddressesSelector), []],
+          [select(walletAddressSelector), '0xxyz'],
+          [call(retrieveSignedMessage), 'some signed message'],
+        ])
+        .put(updateVerifiedAddresses([mockAccount]))
+        .put(endFetchAddressVerification())
+        .run()
+
+      expect(mockFetch).toHaveBeenCalledTimes(1)
+      expect(mockFetch).toHaveBeenCalledWith(
+        `${networkConfig.checkAddressVerifiedUrl}?address=${mockAccount}&clientPlatform=android&clientVersion=0.0.1`,
+        {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            authorization: `Valora 0xxyz:some signed message`,
+          },
+        }
+      )
+    })
+
+    it('skips fetching if address already known', async () => {
+      await expectSaga(fetchAddressVerificationSaga, fetchAddressVerification(mockAccount))
+        .provide([[select(verifiedAddressesSelector), [mockAccount]]])
+        .put(endFetchAddressVerification())
+        .run()
+
+      expect(mockFetch).toHaveBeenCalledTimes(0)
+    })
+
+    it('handles errors gracefully', async () => {
+      mockFetch.mockReject()
+      await expectSaga(fetchAddressVerificationSaga, fetchAddressVerification(mockAccount))
+        .provide([
+          [select(verifiedAddressesSelector), []],
+          [select(walletAddressSelector), '0xxyz'],
+          [call(retrieveSignedMessage), 'some signed message'],
+        ])
+        .put(endFetchAddressVerification())
+        .run()
```

Comment:
should there be some assertion for failure? For example, if someone accidentally removes the mockFetch.mockReject() above the test would still pass?